image_tools/src/showimage.cpp

```diff
@@ -21,13 +21,14 @@
 
 #include <sensor_msgs/msg/image.hpp>
 
+#include <image_tools/options.hpp>
 
 int
 encoding2mat_type(const std::string & encoding)
 {
   if(encoding == "mono8") {
     return CV_8UC1;
-  } else if(encoding == "rgb8") {
+  } else if(encoding == "bgr8") {
```

Comment:
I added a space between the if and the paren and started a new CI job (waiting for the previous one to finish):

http://ci.ros2.org/job/ros2_batch_ci_linux/215/ (not started yet)